aws-sqs-http-connector/go.mod

```diff
@@ -1,12 +1,9 @@
 module github.com/fission/keda-connectors/aws-sqs-http-connector
 
-go 1.12
+go 1.15
 
 require (
-	github.com/aws/aws-sdk-go v1.34.20
-	github.com/fission/keda-connectors/common v0.0.0-20200827103613-b77618fd08e1
+	github.com/aws/aws-sdk-go v1.35.7
```

Comment:
I think you might have done this by mistake. Don't modify aws connector code, please feel to raise separate PR if required.